doc/man1/fipsinstall.pod

```diff
@@ -0,0 +1,148 @@
+=pod
+
+=head1 NAME
+
+openssl-fipsinstall,
+fipsinstall - perform FIPS configuration installation
+
+=head1 SYNOPSIS
+
+B<openssl fipsinstall>
+[B<-help>]
+[B<-in configfilename>]
+[B<-out configfilename>]
+[B<-module modulefilename>]
+[B<-provider_name providername>]
+[B<-section_name sectionname>]
+[B<-verify>]
+[B<-mac_name macname>]
+[B<-macopt>]
+
+B<openssl> I<fipsinstall> [B<...>]
+
+=head1 DESCRIPTION
+
+During FIPS installation this utility is used to run the FIPS modules self tests
+and then generate a FIPS related configuration file that contains:
+
+=over 4
+
+=item - A mac of the FIPS module file.
+
+=item - A status indicator that indicates if the known answer Self Tests (KAT's)
+have successfully run.
+
+=back
+
+This configuration file will then be used each time the FIPS module is loaded
```

Comment:
Should probably be "a FIPS module"...  this is, after all, a command we offer to others to produce their own FIPS module as they see fit, and by implication, we offer them a mechanism to do the integrity checks.

(I *assume* we offer this to others, since it's an `openssl` sub-command.  If that is not the intent, then I agree with @richsalz that this should be a separate program that isn't installed)

Reply:
Well we pass thru the data to the module..
it is up to the module to do whatever it likes with the data.
i.e. It would need to have a MAC algorithm inside its module to run its own integrity check...